File: newgen/playlist/playlist_types_gen.ts
export enum PlaylistItemType {
	Dashboard_by_tag = "dashboard_by_tag",
	Dashboard_by_uid = "dashboard_by_uid",
}

export interface PlaylistItem {
	type: PlaylistItemType;
	value: string;
}

export interface playlist {
	interval: string;
	items: PlaylistItem[];
	name: string;
	xxx: string;
}

File: newgen/playlist/types/playlist_types_gen.go
package types

type PlaylistItem struct {
	Type  PlaylistItemType `json:"type"`
	Value string           `json:"value"`
}

type PlaylistItemType string

const (
	DashboardByTag PlaylistItemType = "dashboard_by_tag"
	DashboardByUid PlaylistItemType = "dashboard_by_uid"
)

type Playlist struct {
	Interval string         `json:"interval"`
	Items    []PlaylistItem `json:"items"`
	Name     string         `json:"name"`
	Xxx      string         `json:"xxx"`
}
